src/sessions/SessionsSection.tsx

```diff
@@ -53,6 +90,25 @@ export const SessionsSection = () => {
           loader={loader}
           ariaLabelKey="session:title"
           searchPlaceholderKey="sessions:searchForSession"
+          searchTypeComponent={
+            <Select
+              width={200}
+              data-testid="filter-session-type-select"
+              isOpen={filterDropdownOpen}
+              className="filter-session-type-select"
+              variant={SelectVariant.single}
+              onToggle={() => setFilterDropdownOpen(!filterDropdownOpen)}
```

Comment:
It looks like `onToggle` provides us with the [new expanded state](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Select/Select.tsx#L110), so we do not have to determine it ourselves.
```suggestion
              onToggle={(isExpanded) => setFilterDropdownOpen(isExpanded)}
```

Reply:
That's a good call!